crates/holochain_zome_types/src/entropy.rs

```diff
@@ -1,17 +0,0 @@
-//! Types for arbitrary data driven by entropy
```

Comment:
Removing this since its related to unused arbitrary code that was already deleted with a previous PR